content/productivity/crawl-planets/plugin.js

```diff
@@ -1,9 +1,22 @@
-import {
-  move
-} from 'https://plugins.zkga.me/utils/queued-move.js';
+const planetTypes = {
+  'Planet': 0,
+  'Asteroid': 1,
+  'Foundry': 2,
+  'Spacetime Rip': 3,
+  'Quasar': 4,
+};
+
+const planetLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+const players = [
+  "0x0000000000000000000000000000000000000000",
+];
+
+const typeNames = Object.keys(planetTypes);
 
 class Plugin {
   constructor() {
+    this.planetType = 1;
     this.minPlanetLevel = 3;
```

Comment:
Let's add layer to allow more flexibility during crawl.

```suggestion
this.minPlanetLevelTo = 3;
this.minPlanetLevelFrom = 0;
this.maxPlanetLevelFrom = 3;
```